spec/requests/core_induction_programme/course_module_spec.rb

```diff
@@ -3,9 +3,79 @@
 require "rails_helper"
 
 RSpec.describe "Core Induction Programme Module", type: :request do
-  it "renders the cip module page" do
-    course_module = FactoryBot.create(:course_module)
-    get "/core-induction-programme/#{course_module.course_year.id}/#{course_module.id}"
-    expect(response).to render_template(:show)
+  let(:course_module) { FactoryBot.create(:course_module) }
+  let(:course_module_url) { "/core-induction-programme/years/#{course_module.course_year.id}/modules/#{course_module.id}" }
+
+  describe "when an admin user is logged in" do
+    before do
+      admin_user = create(:user, :admin)
+      sign_in admin_user
+    end
+
+    describe "GET /core-induction-programme/years/:years_id/modules/module_id" do
+      it "renders the cip module page" do
+        get course_module_url
+        expect(response).to render_template(:show)
+      end
+    end
+
+    describe "GET /core-induction-programme/years/:years_id/modules/module_id/edit" do
```

Comment:
There is a break between tests here, but between some other tests down below. Let's make it consistent :)